packages/itwin/tree-widget/src/test/tree-header/TreeHeader.test.tsx

```diff
@@ -35,6 +35,40 @@ describe("<TreeHeader />", () => {
     sinon.restore();
   });
 
+  it("renders non `enlarged` header", async () => {
```

Comment:
I think same as for buttons this should be tested with e2e test instead of checking `data-iui-size`